### ISSUE TITLE
SSL support not working (likely due to SecureRandom)

### ISSUE THREAD

**Issue by cowboygneox** [betamaxteam/betamax#224]
I'm looking into this, but following my own instructions, I can't get SSL support to work on my Mac. I'm not sure what changed or how it broke. I do know that SSL works just fine if you use the Docker image.

*Work Around:* Use the Docker image for testing.

**Comment by quiqua** [comment]
Worked for me on my Mac:

```
export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"
sudo keytool -importcert -keystore $JAVA_HOME/jre/lib/security/cacerts -file betamax.pem -alias betamax -storepass changeit -noprompt
```

Related: http://stackoverflow.com/questions/6588390/where-is-java-home-on-osx-sierra-10-12-el-captain-10-11-yosemite-10-10

**Comment by cowboygneox** [comment]
@quiqua SDK version?

**Comment by cowboygneox** [comment]
Mine:

```
$ echo $JAVA_HOME
/Library/Java/JavaVirtualMachines/jdk1.8.0_112.jdk/Contents/Home
```

**Comment by quiqua** [comment]
```
java -version
java version "1.8.0_102"
Java(TM) SE Runtime Environment (build 1.8.0_102-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.102-b14, mixed mode)
```

Installed via `brew cask`, but didn't upgrade recently.
`/usr/local/Caskroom/java/1.8.0_102-b14 (227.5M)`

**Comment by cowboygneox** [comment]
Actually, once I actually installed the cert into `8u112`, it worked. However, I still can't get `8u111` to work...probably why `8u112` exists...

**Comment by cowboygneox** [comment]
Of all of the problems I've had in my life, `SecureRandom` appears to be the worst, and I've broken my knee and had two surgeries on it.

**Comment by cowboygneox** [comment]
If you look in the logs of the latest failed Travis build, you see a lot of this:

```
Failed to generate a seed from SecureRandom within 3 seconds. Not enough entrophy?
```

That's likely the issue I'm running into locally.

**Comment by cowboygneox** [comment]
My `8u111` is now working as well. I'm increasingly confident it's a `SecureRandom` issue.

**Comment by cowboygneox** [comment]
I'm going to explore this a bit more, but based on the [release notes for JDK 8u112](http://www.oracle.com/technetwork/java/javase/8u112-relnotes-3124973.html), I think I'm going to make the recommended JDK for Betamax to be `8u112`.

**Comment by cowboygneox** [comment]
I'm also probably going to suggest this change: https://docs.oracle.com/cd/E13209_01/wlcp/wlss30/configwlss/jvmrand.html.

**Comment by cowboygneox** [comment]
Resolved by #225.
